drivers/hmis/app/graphql/mutations/ce/mark_units_available.rb

```diff
@@ -37,6 +37,10 @@ def resolve(unit_ids:)
         Hmis::Ce::Opportunity.import!(opportunities)
       end
 
+      job_class = Hmis::CandidatePoolBuilderJob
+      job_name = job_class.name
+      Delayed::Job.enqueue job_class.new unless Delayed::Job.queued?(job_name) || Delayed::Job.running?(job_name)
```

Comment:
I think this should include the queue:

```suggestion
      Delayed::Job.enqueue(job_class.new, queue: ENV.fetch('DJ_LONG_QUEUE_NAME', :long_running)) unless Delayed::Job.queued?(job_name) || Delayed::Job.running?(job_name)
```